docs/modules/ROOT/pages/relay.adoc

```diff
@@ -160,6 +160,228 @@ All private keys are stored in the AWS Key Management Service. Keys are generate
+
+Here is an example of getting a JWT Token in nodejs using `amazon-cognito-identity-js`. The Username/Password pair is your ApiKey/ApiSecret pair. PoolId and PoolClientId are always the same.
+
+```JavaScript
+import { AuthenticationDetails, CognitoUserPool, CognitoUser } from 'amazon-cognito-identity-js';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+global.fetch = require('node-fetch').default;
+
+export const RelayerPoolId = () => process.env.RELAY_POOL_ID || 'us-west-2_iLmIggsiy'; // always the same
+export const RelayerPoolClientId = () => process.env.RELAY_POOL_CLIENT_ID || '1bpd19lcr33qvg5cr3oi79rdap'; // always the same
+
+type UserPass = { Username: string; Password: string };
+
+export async function authenticate(authenticationData: UserPass): Promise<string> {
+  const authenticationDetails = new AuthenticationDetails(authenticationData);
+  const poolData = {
+    UserPoolId: RelayerPoolId(),
+    ClientId: RelayerPoolClientId(),
+  };
+
+  const userPool = new CognitoUserPool(poolData);
+  const userData = { Username: authenticationData.Username, Pool: userPool };
+  const cognitoUser = new CognitoUser(userData);
+
+  return new Promise((resolve, reject) => {
+    cognitoUser.authenticateUser(authenticationDetails, {
+      onSuccess: function (session) {
+        const token = session.getAccessToken().getJwtToken();
+        resolve(token);
+      },
+      onFailure: function (err) {
+        console.error(`Failed to get a token for the API key ${authenticationData.Username}`, err);
+        reject(err);
+      },
+    });
+  });
+}
+```
+
+=== Making Requests
+
+Once you get a JWT Token you can make requests to Defender API. A request requires ApiKey, JWT Token, optionally a payload, and API URL. Set $KEY, $TOKEN to the values of API Key and JWT Token accuired before. $END_POINT can be either `txs` or `sign`.
+
+```bash
+API_URL='http://api.defender.openzeppelin.com/'
+
+curl \
+  -H 'Accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -H "X-Api-Key: $KEY" \
+  -H "Authorization: Bearer $TOKEN" \
+    "$API_URL/$END_POINT"
+```
+
+=== Txs Endpoint
+
+==== Send Transaction
+To send a transaction to Ethereum blockchain submit a POST request with a desired payload. Payload format:
+
+```TypeScript
+export type Address = string;
+export type BigUInt = string | number;
+export type Hex = string;
+export type Speed = 'safeLow' | 'average' | 'fast' | 'fastest';
+
+export interface SendTransactionRequest {
+  to: Address;
+  value?: BigUInt; // optional
+  data?: Hex; // optional 
+  speed?: Speed; // optional
+  gasLimit: BigUInt;
+}
+```
+An example of the request:
+
+```bash
+DATA='{ "to": "0x179810822f56b0e79469189741a3fa5f2f9a7631", "value": "1", "speed": "fast", "gasLimit": "21000" }'
+
+curl \
+  -X POST \
+  -H 'Accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -H "X-Api-Key: $KEY" \
+  -H "Authorization: Bearer $TOKEN" \
+  -d "$DATA" \
+    "$API_URL/txs"
```

Comment:
Let's make the endpoint itself more prominent. It's the most important piece of info users need, and it's a bit buried within the curl command.